test/evergreen.yml

```diff
@@ -173,13 +173,16 @@ functions:
       script: |
         set -o errexit
         set -o verbose
+        dir=$(pwd)
+        cd wiredtiger
         mongo_repo=https://github.com/mongodb/mongo
-        branch=${branch_name}
+        branch=$(git branch --contains | grep -e "mongodb" -e "develop")
```

Comment:
Why can't we set `$branch_name` differently across runs instead of auto-detecting it like this? (Either way, I think this line justifies a comment explaining why we have to detect the mongo branch based on WT's checked-out branch.)

Reply:
$branch_name is defined by the evergreen project configuration, it automatically gets the branch name from your git. It's something that can't be easily configured by developers unfortunately. I have added a comment let me know if it clear enough

Reply:
Got it - thank you. New comment looks good, too.